packages/core/src/app/shipping/MultiShippingFormV2.test.tsx

```diff
@@ -8,7 +8,11 @@ import userEvent from '@testing-library/user-event';
 import React from 'react';
 
 import { ExtensionProvider } from '@bigcommerce/checkout/checkout-extension';
-import { createLocaleContext, LocaleContext } from '@bigcommerce/checkout/locale';
+import {
+    createLocaleContext,
+    LocaleContext,
+    LocaleContextType,
```

Comment:
`LocaleContextType` is used on L35, but it was never imported until now.
I feel we missed some config here. 🤦🏻‍♂️ Will check it tomorrow.
cc @richapatel1510 @bc-mackxu